modules/core/src/command_line_parser.cpp

```diff
@@ -9,6 +9,8 @@ static const char* noneValue = "<none>";
 
 static String cat_string(const String& str)
 {
+    if (str.length() == 0)
+        return String("");
     int left = 0, right = (int)str.length();
     while( left <= right && str[left] == ' ' )
```

Comment:
> It is better to replace `<=` to `<` instead.

I didn't want to change any existing code because I figured it's already working as it is.

> Also please remove unnecessary commits from PR.

Sorry, but I don't know how to do that. I only do pull requests once or twice a year and I always completely screw up my local repository. It's pretty hopeless at this point.